internal/npo-ext.owl

```diff
@@ -3031,6 +3051,84 @@
     </owl:Class>
 
 
+    <!-- http://purl.enanomapper.org/onto/ENM_0000064 -->
+
+    <owl:Class rdf:about="http://purl.enanomapper.org/onto/ENM_0000064">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0000070"/>
+        <rdfs:isDefinedBy>Technique used to determine the surface area of solid particles.</rdfs:isDefinedBy>
+        <rdfs:label xml:lang="en">Brunauer–Emmett–Teller analysis</rdfs:label>
+    </owl:Class>
+
+
+    <!-- http://purl.enanomapper.org/onto//ENM_0000074 -->
+
+    <owl:Class rdf:about="http://purl.enanomapper.org/onto//ENM_0000074">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0000070"/>
+        <rdfs:isDefinedBy>Optical system that uses laser diffraction to measure particle size.</rdfs:isDefinedBy>
+        <rdfs:label xml:lang="en">Mastersizer</rdfs:label>
+    </owl:Class>
+    
+    
+    <!-- http://purl.enanomapper.org/onto//ENM_0000075 -->
+
+    <owl:Class rdf:about="http://purl.enanomapper.org/onto//ENM_0000075">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0000070"/>
+        <rdfs:label xml:lang="en">Malvern Zetasizer</rdfs:label>
+    </owl:Class>
+
+
+    <!-- http://purl.enanomapper.org/onto/ENM_0000065 -->
+ 
+    <owl:Class rdf:about="http://purl.enanomapper.org/onto/ENM_0000065">
+        <rdfs:subClassOf rdf:resource="http://purl.bioontology.org/ontology/npo#NPO_1405"/>
+        <rdfs:isDefinedBy>Particle sizing technique based on Brownian motion and scattered light.</rdfs:isDefinedBy>
+        <rdfs:label xml:lang="en">Nanoparticle Tracking Analysis</rdfs:label>
+    </owl:Class>
+    
+    
+    <!-- http://purl.enanomapper.org/onto/ENM_0000073 -->
+    
+    <owl:Class rdf:about="http://purl.enanomapper.org/onto/ENM_0000073">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/CHEBI_51086"/>
+        <rdfs:isDefinedBy>A liquid used to extract a substance from another material.</rdfs:isDefinedBy>
+        <rdfs:label xml:lang="en">extractant</rdfs:label>
+    </owl:Class>
+    
+
+    <!-- http://purl.enanomapper.org/onto/ENM_0000094 -->
+
+    <owl:Class rdf:about="http://purl.enanomapper.org/onto/ENM_0000094">
+        <rdfs:subClassOf rdf:resource="http://purl.org/obo/owl/FIX#FIX_0000053"/>
```

Comment:
Is there a reason to prefer FIX to CHMO ? (e.g. CHMO_0001000 chromatography )

Reply:
we do not know what the FIX is here. the URL did not resolve for me, but it seems an ontology known to BioPortal: http://bioportal.bioontology.org/ontologies/FIX/

However, it cannot be an superclass, because we do not use FIX (at this moment).